internal/storage/storage_test.go

```diff
@@ -84,12 +101,26 @@ func TestNewBackend(t *testing.T) {
 	p, err := NewBackend(testConf)
 	assert.Nil(t, err, "Backend posix failed")
 
+	var buf bytes.Buffer
+	log.SetOutput(&buf)
+
+	testConf.Type = sftpType
+	sf, err := NewBackend(testConf)
+	assert.Nil(t, err, "Backend sftp failed")
+	assert.NotZero(t, buf.Len(), "Expected warning missing")
+
+	// update host key from server for later use
+	rgx := regexp.MustCompile(`\\\"(.*?)\\"`)
+	testConf.SFTP.HostKey = strings.Trim(rgx.FindString(buf.String()), "\\\"")
+	buf.Reset()
```

Comment:
This can be tested against a mocked server see [here](https://medium.com/@metarsit/ssh-is-fun-till-you-need-to-unit-test-it-in-go-f3b3303974ab) for some inspiration. Other similar variants can be found elsewhere

Reply:
I initially tried to avoid having the live sftp server but got stuck with sftp mocking and didnot want to spend too much time on it (especially after I noticed that we have a live mq and db for the testsuite anyway). But I'll give it another go :)

Reply:
All go testing is now done with a mock sftp server with no live server dependency :-).